src/Skeleton/Telepes.java

```diff
@@ -150,24 +143,21 @@ boolean NyerEllenoriz(Epitheto e) {
 
 
 	void AnyagVisszatesz(Nyersanyag n){
-	    aszteroida.addMag(n);
+	    aszteroida.AddMag(n);
 
 	}
 
-	void AddRakter(Nyersanyag n){
-		Rakter.add(n);
+	void AddRakter(Nyersanyag n){		
+		//rakter.add(n);
```

Comment:
Ezzel mi legyen?
Itt kéne a type check, nem tudom mit mondtunk végül rá

Reply:
Mindenképp valamit csinálj vele merge előtt, nehogy baj legyen belőle...

Reply:
ok... Majd megoldja, aki het5 branchbe nem forduló kódot húzott be.